docs/CONTRIBUTIONS.md

```diff
@@ -4,10 +4,41 @@
 
 During the project some contributions to outside open source projects have been made. These contributions are listed below.
 
-## `deno-swagger-doc` Module
+## Publications
 
-Fixed an old dependency function call. [See here](https://github.com/singhcool/deno-swagger-doc/pull/11) for more information.
+### Jitsi API Client
 
-## `Trex` Package Manager
+Published the Jitsi API Client Module on [deno.land](https://deno.land/x/jitsi_api_client). This module represents a convenience wrapper for the low level Jitsi API.
 
-Reported an issue with `trex` commands failing when executed in GitHub Actions. [See here](https://github.com/crewdevio/Trex/issues/123) for more information.
+### Jimmi API Client
+
+Published the Jimmi API Client Module on [deno.land](https://deno.land/x/jimmi_api_client). This module represents a convenience wrapper for the Jimmi backend REST-API.
+
+### Jimmi
+
+Published the Jimmi Module on [deno.land](https://deno.land/x/jimmi). This module represents the Jimmi application for listening to music in a Jitsi conference call 🎶.
+
```

Comment:
Do we actually publish Jimmi on deno.land? It is packaged as binary, portable archive and container image. Not sure if publishing on deno.land will add value :thinking: